test/test-v8debugapi.js

```diff
@@ -319,9 +321,39 @@ describe('v8debugapi', function() {
     '{f: process.env}',
     '1,2,3,{f:2},4',
     'A[this?this:1]',
-    '[1, 2, 3, 4, x == 1, x === 1, null, undefined]'
+    '[1, 2, 3, 4, x == 1, x === 1, null, undefined]',
+    '[0].values',
+    '[][0]',
+    '[0][' + MAX_INT + ']',
+    '"𠮷".length + (5| "𠮷")',
+    '/ٹوٹ بٹوٹ کے دو مُرغے تھے/',
   ]);
 
+  if (semver.satisfies(process.version, '>=4.0')) {
```

Comment:
Could we also have a check that unsupported ES6 features are handled gracefully on versions of V8 that do not support them?